js/buttons.html5.js

```diff
@@ -865,19 +865,20 @@ DataTable.ext.buttons.copyHtml5 = {
 			textarea[0].select();
 
 			try {
-				document.execCommand( 'copy' );
+				var successful = document.execCommand( 'copy' );
 				hiddenDiv.remove();
```

Comment:
Should this be inside the if statement, i.e. only remove if the copy to clipboard succeeds?